addons/point_of_sale/static/src/js/screens.js

```diff
@@ -314,7 +313,12 @@ var ScaleScreenWidget = ScreenWidget.extend({
             // add product *after* switching screen to scroll properly
             self.order_product();
         });
+        this._read_scale();
+    },
 
+    _read_scale: function() {
+        var self = this;
+        var queue = this.pos.proxy_queue;
```

Comment:
i suppose this change is done for an easier override in another addon, i suppose this should be split in another commit and explained

Reply:
Indeed. It has its own commit with a message explaining the reason for the change.